pytorch_pfn_extras/onnx/export_testcase.py

```diff
@@ -137,6 +138,9 @@ def _export_util(
             except checker_error:
                 if enable_onnx_checker:
                     raise
+                if return_output:
+                    # Re-run the model to obtain the output.
+                    return model(*args)
```

Comment:
Do you still need the outputs even when the export fails?

Reply:
Thanks for the question! Historically, `torch.onnx.export` had a kwarg `enable_onnx_checker` which turns off ONNX checker. As the kwarg is removed, we are emulating the behavior here by catching `torch.onnx.utils.ONNXCheckerError`. The emulation was working good for `ppe.onnx.export` as it does not need outputs but was broken for `ppe.onnx.export_testcase`. Without this PR, `test_data_set_0/output_0.pb` will be missing if we export ONNX-incompliant ONNX.

Reply:
Also please note `torch.onnx` ensures model export fails due to `CheckerError`:
```
      CheckerError: If the ONNX checker detects an invalid ONNX graph. Will still export the model to the file ``f`` even if this is raised.
```